docs/getting-started/provider-playbook/technology-providers-implementation/ed-fi-api-fundamentals/enumerations-descriptors.md

```diff
@@ -0,0 +1,88 @@
+```
+
+   ...and _must not_ be sent as
+
+```text
+uri://ed-fi.org/AcademicSubjectDescriptor#English%20Language%20Arts
+```
+
+## Descriptor API Resources
+
+Many APIs (including that on the Ed-Fi ODS/API) provide lookup features for Descriptors. This allows an API to load and inspect all possible values for a particular enumeration. The API resource format for a Descriptor has this format:
+
+```json
+  {
+    "id": "47721c744e566176ad035dc8e119a2fb",
+    "codeValue": "English Language Arts",
+    "description": "English Language Arts",
+    "namespace": "uri://ed-fi.org/AcademicSubjectDescriptor",
+    "shortDescription": "English Language Arts"
+  }
+```
+
+Note that a Resource ID ("id") is provided here, consistent with other Ed-Fi REST API resources.
+
+## Ed-Fi, Provider, and Agency Values: What to Expect and What to Use
+
+Interoperability is complex, and enumeration values are part of this complexity. While it seems like the goal should be to coordinate the whole K-12 ecosystem to employ use a single set of values for each core K-12 concept, the reality is that education agencies and technology providers must adapt values to local needs and conditions.
+
+For agencies, such adaptation often reflects important local programs and priorities. Further, use of some sets of values might be mandated by (or reflect) state or local policy. These make adoption of a global set problematic.
+
+For their part, technology providers often have highly refined sets of values that reflect years worth of process and analytic investment.
+
+As a result, users of Ed-Fi APIs should expect to see enumeration sets with mixed namespaces.
+
+### Use Case-Specific Information
+
+Ed-Fi API specifications and certifications capture current community expectations as to what enumeration sets to use. Specifications in use can be found here:
+
+[Ed-Fi Data Exchange Standards Home](https://edfi.atlassian.net/wiki/spaces/EFDS/overview)
+
+Specifications in a "Draft" Request for Comment (RFC) state (which are also commonly found) can be found here:
+
+[Ed-Fi RFC Home](https://edfi.atlassian.net/wiki/spaces/EFDSRFC/overview)
+
+Certifications for these specifications and others are listed here:
+
+[Available Certifications](https://edfi.atlassian.net/wiki/spaces/EDFICERT/pages/23692586/Available+Certifications)
+
+### Example: Assessment Outcomes API
+
+As an example, let's look at the Assessment Outcomes API.
+
+In this API (which is defined in [ED-FI RFC 15 - ASSESSMENT OUTCOMES API](https://edfi.atlassian.net/wiki/spaces/EFDSRFC/pages/25362990/ED-FI+RFC+15+-+ASSESSMENT+OUTCOMES+API)), there are no requirements as to what enumeration sets are required. However, in the certification for this API, only certain enumerations are permitted to be tech-provider-supplied and others must be Ed-Fi-governed values.  There are currently three major versions of the Data Standard available so look at these locations for examples:
```

Comment:
This is an example of a very dangerous link. Someone who follows this and tries to build an Assessment Outcomes API will be building on Data Standard 2.2, which was deprecated years ago.

We should change to this link, which is at least on Data Standard 3 and not so far off as the Data Standard 2 doc.

```suggestion
In this API (which is defined in [ED-FI RFC 15 - ASSESSMENT OUTCOMES API](https://edfi.atlassian.net/wiki/spaces/EFDS/pages/17727736/Ed-Fi+Assessment+Outcomes+API+for+Suite+3)), there are no requirements as to what enumeration sets are required. However, in the certification for this API, only certain enumerations are permitted to be tech-provider-supplied and others must be Ed-Fi-governed values.  There are currently three major versions of the Data Standard available so look at these locations for examples:
```

Reply:
updated.